package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@sapphire/cli",
-	"version": "1.3.1",
+	"version": "1.4.0",
```

Comment:
Revert this. We use automated conventional commit based version bumping.

Reply:
ok

Reply:
done